astropy/coordinates/builtin_frames/equatorial.py

```diff
@@ -64,7 +64,7 @@ class TETE(BaseRADecFrame):
     with local apparent sidereal time to calculate the apparent (TIRS) hour angle.
 
     For more background on TETE, see the references provided in the
-    :ref:`astropy-coordinates-seealso` section of the documentation.
+    :ref:`astropy:astropy-coordinates-seealso` section of the documentation.
```

Comment:
These `astropy:astropy` references are unfortunate.  I won't ask you to change them, but I wonder why those labels contained "astropy-" in the first place.  Seems redundant.  We can deal with that in a follow-up, if at all.

Reply:
Agreed. I didn't like those either. And they're now doubly redundant.